CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 2.32.0-rc.0 / 2021-12-01
+
+This list of changes is relative to [v2.32.0-beta.0](https://github.com/prometheus/prometheus/releases/tag/v2.32.0-beta.0), which was a special 2.32 beta release to pre-publish the Prometheus Agent features.
+
+* [FEATURE] UI: Adapt web UI for Prometheus Agent mode. #9851.
+* [ENHANCEMENT] Linode SD: Tune API request page sizes. #9779
+* [ENHANCEMENT] TSDB: Check postings bytes length when decoding. #9766
+* [BUGFIX] TSDB: Fix panic when checkpoint directory is empty. #9687
+* [BUGFIX] Web: Fix a potential goroutine leak by channel. #8965
+* [BUGFIX] TSDB: Fix panic, out of order chunks, and race warning during WAL replay. #9856
+* [BUGFIX] TSDB: Fix and improve the `Seek()` method of various iterators. #9878
+* [BUGFIX] UI: Correctly render links for targets with IPv6 addresses that contain a Zone ID. #9853
+* [BUGFIX] Promtool: Fix checking of `authorization.credentials_file` field. #9883
```

Comment:
This also fixed the checking of the deprecated but still usable `bearer_token_file`field.

Reply:
Thanks, updated!